docs/dev/connectors/kafka.md

```diff
@@ -405,8 +385,7 @@ Consider setting appropriate [idelness timeouts]({{ site.baseurl }}/dev/event_ti
  
 ## Kafka Producer
```

Comment:
Shall we update the `Migrating Kafka Connector from 0.11 to universal` section? I think the approach recommended there is not valid (at least for the producer).

Reply:
Ah yeah, I actually changed https://issues.apache.org/jira/browse/FLINK-19193 to make it work better for the Kafka connector. 👌